### PR TITLE
Update readme with quickstart content

### PR DESCRIPTION
**Pull request by bdemers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,75 @@
...
+|----------|---------|---------|
+| okta.oauth.issuer     | N/A | [Authorization Server](/docs/how-to/set-up-auth-server.html) issuer URL, i.e.: https://{yourOktaDomain}.com/oauth2/default |
+| okta.oauth.clientId   | N/A | The Client Id of your Okta OIDC application |
+| okta.oauth.audience   | api://default | The audience of your [Authorization Server](/docs/how-to/set-up-auth-server.html) |
+| okta.oauth.scopeClaim | scp | The scope claim key in the Access Token's JWT |
+| okta.oauth.rolesClaim | groups | The claim key in the Access Token's JWT that corresponds to an array of the users groups. |
+
+## Create a Controller 
+
+The above client makes a request to `/hello_oauth`, we simply need to create a `Controller` to handle the response:
```
Comment (mraible) [inline README.md]: I'd change the URL to hello-oauth.

Comment (mraible) [inline README.md]: Suggest changing "we" to "you"

File: README.md [hunk 2]
```diff
@@ -1,35 +1,75 @@
...
+```bash
+$ curl http://localhost:8080/hello_oauth \
+   --header "Authorization: Bearer ${accessToken}"
+```
+The result should look something like:
 ```
+Hello, joe.coder@example.com
+```
+
+Okta's Spring Security integration will [parse the JWT access token](/blog/2017/06/21/what-the-heck-is-oauth#oauth-flows) from the HTTP request's `Authorization: Bearer` header value.
```
Comment (mraible) [inline README.md]: Use full URL to blog post

File: README.md [hunk 3]
```diff
@@ -1,35 +1,75 @@
...
+
+Check out a minimal example that uses the [Okta Signin Widget and JQuery](examples/siw-jquery) or this [blog post](https://scotch.io/@mraible/build-a-secure-notes-application-with-kotlin-typescript-and-okta).
```
Comment (mraible) [inline README.md]: I'd include "this" in the blog post link.

File: README.md [hunk 4]
```diff
@@ -1,35 +1,75 @@
...
+```
+
+Okta's Spring Security integration will [parse the JWT access token](/blog/2017/06/21/what-the-heck-is-oauth#oauth-flows) from the HTTP request's `Authorization: Bearer` header value.
+
+Check out a minimal example that uses the [Okta Signin Widget and JQuery](examples/siw-jquery) or this [blog post](https://scotch.io/@mraible/build-a-secure-notes-application-with-kotlin-typescript-and-okta). 
+
+
+# Extra Credit
+
+Want to build this project?, just clone it out and run `mvn install`
```
Comment (mraible) [inline README.md]: Start a new sentence after comma. You might want to include the git clone, cd, and mvn commands in a code block.

**Comment by bdemers** [comment]
Thanks @mraible !

File: README.md [hunk 5]
```diff
@@ -1,35 +1,83 @@
-okta-spring-security
-====================
+Okta Spring Boot Starter
+========================
 
-Still some work to be done on this README.
+Okta's Spring Boot Starter will enable your Spring Boot application to work with an Okta access token via an OAuth implicit flow.  Jump to our [quickstart](https://developer.okta.com/quickstart/#/angular/java/spring) to see how to configure various clients or follow along below and to use curl.
```
Comment (mraible) [inline README.md]: Is the link to the Quickstart correct? The landing page only seems to show dependencies and properties for this starter.

- "follow along below and" should be "follow along below".

File: README.md [hunk 6]
```diff
@@ -1,35 +1,83 @@
...
+}
 ```
 
-Pre-commit, to include license header checking, PMD, and Findbugs (including find-sec-bugs) use the `ci` profile:
-``` bash
-# can only be run from the root of the project
-mvn clean install -Pci
+## That's it!
+
+To test things out we can use curl:
```
Comment (mraible) [inline README.md]: Change "we" to "you"

File: README.md [hunk 7]
```diff
@@ -1,35 +1,83 @@
...
+
+Check out a minimal example that uses the [Okta Signin Widget and JQuery](examples/siw-jquery) or [this blog post](https://scotch.io/tutorials/build-a-secure-notes-application-with-kotlin-typescript-and-okta). 
+
+
+# Extra Credit
+
+Want to build this project? 
 
-- Requires a custom Authorization Server, (instructions to be added soon)
+Just clone it out and run:
```
Comment (mraible) [inline README.md]: "clone it and run" instead of "clone it out and run"

Comment (bdemers) [inline README.md]: Wow... I'm not even sure how I wrote that one.

**Pull request merged**